bin/scripts/export-data/setting_limits_and_real_values.js

```diff
@@ -0,0 +1,356 @@
+/**
+ *  Description: Creates JSON file of events, views, and custom user properties, their segments, and segment values.
+ *  Server: countly
+ *  Path: $(countly dir)/bin/scripts/export-data
+ *  Command: node setting_limits_and_real_values.js
+ */
+
+const { ObjectId } = require('mongodb');
+const fs = require('fs');
+const crypto = require('crypto');
+const common = require('../../../api/utils/common.js');
+const pathToFile = './'; // path to save json files
+const WriteStream = fs.createWriteStream(pathToFile + `setting_limits_and_values.json`);
+const pluginManager = require('../../../plugins/pluginManager.js');
+const app_list = []; // valid app_ids here. If empty array passed, script will process all apps.
+const DEFAULTS = {
+    event_limit: 500,
+    event_segment_limit: 100,
+    event_segment_value_limit: 1000,
+    view_limit: 50000,
+    view_name_limit: 128,
+    view_segment_limit: 100,
+    view_segment_value_limit: 10,
+    custom_prop_limit: 20,
+    custom_prop_value_limit: 50,
+};
+
+Promise.all([pluginManager.dbConnection("countly"), pluginManager.dbConnection("countly_drill")]).then(async function([countlyDb, drillDb]) {
+    console.log("Connected to databases...");
+    try {
+        var all_results = [];
+        const apps = await getAppList({db: countlyDb});
+        if (!apps || !apps.length) {
+            return close();
+        }
+        else {
+            // GETTING DATA FOR SET LIMITS FOR EVENTS, VIEWS, AND CUSTOM PROPERTIES
+            var pluginsCollectionPlugins = await countlyDb.collection("plugins").findOne({"_id": 'plugins'});
+
+            // LOOP APPS FOR EACH REQUIREMENT
+            for (let i = 0; i < apps.length; i++) {
+                var app = apps[i];
+                console.log(i + 1, ") Processing app:", app.name);
+
+                try {
+                    var app_results = { "App Name": app.name },
+                        defaultVal,
+                        realVal,
+                        currentVal;
+
+                    // GETTING DATA PER APP
+                    var eventsCollectionPerApp = await countlyDb.collection("events").findOne({"_id": ObjectId(app._id)});
```

Comment:
[{"$match":{"_id": ObjectId(app._id)}}, {"$project":{"$_id":"$_id","eventcount":{"$size":"$list"}}}]
(wrap $list also in $ifNull in case, validate syntax)